docs/2.8.0/docs/canton/architecture/overview.rst

```diff
@@ -563,101 +528,49 @@ A canonical example is a participant run by a trusted market operator.
-participant, not even in an encrypted form. Furthermore, depending on
-the confirmation policy, some informees are marked as
-confirmers. In addition to distributing the transaction projections
-among participants, the submitter informs the mediator about the
-informees and confirmers of the transaction.
-
-**Validity and Confirmations Responses:** Each informee of a
-requested transaction performs local checks on the validity of its
-visible subtransaction. The informees check that their provided
-projection conforms to the Daml semantics, and the ledger authorization
-model. Additionally, they check whether the request conflicts with an
-earlier request that is accepted or is not yet decided. Based on this,
-they send their responses (one for each of their views), together with
-the informee information for their projection, to the
-mediator.
-When the other participants or domain entities do not behave according to the protocol (for example, not sending
-timely confirmation responses, or sending malformed requests), the transaction processing component raises alarms.
-
-**Confirmation Result Processing**. Based on the result message
-from the mediator, the transaction component commits or aborts
-the requested transaction.
-
-Sequencer Client
-~~~~~~~~~~~~~~~~
+A ledger application uses the ledger API (at the top of the diagram) to send commands to a participant and
+to receive the corresponding events.
+The command submission service receives a command, parses it, and performs some basic validation.
+Next, the command is submitted to DAMLe (DAML engine), which translates it to a transaction;
+a command consists only of a root node whereas a transaction also recursively contains all consequences of all exercise actions.
+Then, the domain router chooses a domain that is suitable for executing the transaction.
 
-The sequencer client handles the connection to the sequencer,
-ensures in-order delivery and stores the cryptographic proofs of
-authenticity for the messages from the sequencer.
+The transaction processor translates the transaction to a confirmation request;
+in particular, it computes the view decomposition, embeds the transaction into a Merkle tree, and
+creates different envelopes tailored to the different members validating the request.
+It uses the sequencer client to send the confirmation request to the mediator and all participants involved in the transaction.
 
-Identity Client
-~~~~~~~~~~~~~~~
+The transaction processor also uses the sequencer client to receive confirmation requests from the domain,
+to send mediator responses, and to receive the result messages from the mediator.
 
-The identity client handles the messages coming from the domain topology manager, and verifies the validity of the
-received identity information changes (for example, the validity of public key delegations).
+The multi domain event log stores for every request if has been committed vs. rolled back.
```

Comment:
Words missing here? Stores *what* for every request? "if has been committed" -- if *it* has been committed? 

Also "multi domain" needs a hyphen: "multi-domain"

Maybe it's this:
```suggestion
The multi-domain event log stores every committed request (but not rolled-back requests).
```

Reply:
I couldn't apply this as is, because is also stores rollbacks.

Reply:
Yep, I said "maybe" because I wasn't sure I understood... 🙂 Updated my suggestion.